packages/wrangler/src/wasi.ts

```diff
@@ -0,0 +1,70 @@
+import { writeFile } from "node:fs/promises";
+import path from "node:path";
+import tmp from "tmp-promise";
+
+// TODO:
+// - wrangler --local is broken due to different bundling behavior between 'ModuleCollector' for
+//   Remote and miniflare cli for Local
+
+function makeEntrypoint(wasmPath: string, modulePath: string): string {
+  const name = path.basename(wasmPath);
+
+  // Allow setting cli arguments by putting WRANGLER_WASM_ARGS=foo=bar,baz in wrangler
+  // environment, e.g., WRANGLER_WASM_ARGS=foo=bar,baz -> hello.wasm foo=bar baz
+  const argsPrefix = "WRANGLER_WASM_ARGS";
```

Comment:
```suggestion
  const argsPrefix = "WRANGLER_WASI_ARGS";
```
Should we make this more specific to WASI?

Reply:
I don't have a strong preference (but I don't think WASM is ambiguous because in what other context would args/env variables be passed to wasm?)

I would consider this config an experimental placeholder, once we get folks playing around with this I'd like to meet and discuss the appropriate cli interface (ideally we get close to `wasmtime run`)

Reply:
I actually think we should avoid using the WASI terminology where possible